test/unit.test.js

```diff
@@ -19,11 +19,27 @@ var Converter = require('../src/lib.js'),
 	});
 
 	describe('validator should', function () {
-		it('indicate valid curl', function () {
+		it('indicate valid curl', function (done) {
 			expect(validate('curl helloworld').result).to.be(true);
 
 			var result = validate('invalid curl');
 			expect(result.result).to.be(false);
+
+			done();
 		});
 	});
 
+	describe('sanitizeArgs function should', function () {
+		it('remove all unnecessary $ from curl string', function (done) {
+			let curlString = `curl -i -s -k  -X $'POST' \
+			-H $'Host: example.com.br' -H $'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:68.0) Gecko/20100101 Firefox/68.0'\
+			$'https://example.com.br/login.html'`
```

Comment:
Shouldn't we check for this $ being getting removed too as we are checking for all the $ signs ?
There doesn't seem to be any test for this.

Reply:
We don't need to provide the arguments curl, -i, -s etc. As mentioned by @abhijitkane , we should be using the simplest input for testing the exact thing you are checking for.